test/fuzz/utility.h

```diff
@@ -11,6 +11,8 @@
 namespace Envoy {
 namespace Fuzz {
 
+std::string kEmptyString = "";
```

Comment:
nit: we have EMPTY_STRING defined somewhere, can you use that?

Reply:
I can! I didn't use it before because of some type mismatching, but the fix here is to make the return value of the mock method's match ConnectionInfo's and return const std::string&'s (and thus enable me to use EMPTY_STRING). Thanks!